Sources/Rendering/Core/AxesActor/index.js

```diff
@@ -65,34 +69,38 @@ function vtkAxesActor(publicAPI, model) {
   publicAPI.setMapper(_mapper);
 
   publicAPI.update = () => {
+    let direction;
+    direction = [model.xAxisInvert ? -1 : 1, 0, 0];
     const xAxis = vtkArrowSource
-      .newInstance({ direction: [1, 0, 0], ...model.config })
+      .newInstance({ direction, ...model.config, ...model.xConfig })
       .getOutputData();
     if (model.config.recenter) {
       centerDataSet(xAxis);
     } else {
-      shiftDataset(xAxis, 0);
+      shiftDataset(xAxis, 0, model.xAxisInvert);
     }
     addColor(xAxis, ...model.xAxisColor);
 
+    direction = [0, model.yAxisInvert ? -1 : 1, 0];
     const yAxis = vtkArrowSource
-      .newInstance({ direction: [0, 1, 0], ...model.config })
+      .newInstance({ direction, ...model.config, ...model.yConfig })
       .getOutputData();
     if (model.config.recenter) {
       centerDataSet(yAxis);
     } else {
-      shiftDataset(yAxis, 1);
+      shiftDataset(yAxis, 1, model.yAxisInvert);
     }
 
     addColor(yAxis, ...model.yAxisColor);
 
+    direction = [0, 0, model.zAxisInvert ? -1 : 1];
```

Comment:
if `vtkArrowSource` got `invert: true` (because `model.zConfig` contains `invert: true`, then you do not need to flip the `direction` anymore.